pyro/analysis/gauss_diffusion_compare.py

```diff
@@ -58,7 +58,7 @@ def process(file):
     rmin = 0
     rmax = np.sqrt(myg.xmax**2 + myg.ymax**2)
 
-    nbins = np.int(np.sqrt(myg.nx**2 + myg.ny**2))
+    nbins = np.int64(np.sqrt(myg.nx**2 + myg.ny**2))
 
```

Comment:
it looks like the suggested way to do this conversion is to use `.astype(np.int64)`:
https://numpy.org/doc/stable/reference/generated/numpy.ndarray.astype.html